packages/create/package.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "@sap-ux/create",
+    "description": "SAP Fiori tools module to add or remove features",
+    "version": "0.0.1",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/SAP/open-ux-tools.git",
+        "directory": "packages/create"
+    },
+    "bugs": {
+        "url": "https://github.com/SAP/open-ux-tools/issues?q=is%3Aopen+is%3Aissue+label%3Abug+label%3Acreate"
+    },
+    "license": "Apache-2.0",
+    "bin": {
+        "create-fiori": "./dist/index.js"
+    },
+    "engines": {
+        "pnpm": ">=6.26.1 < 7.0.0 || >=7.1.0",
+        "node": ">= 14.16.0 < 15.0.0 || >=16.1.0 < 17.0.0 || >=18.0.0 < 19.0.0"
+    },
+    "scripts": {
+        "build": "tsc --build",
+        "clean": "rimraf dist coverage *.tsbuildinfo",
+        "format": "prettier --write '**/*.{js,json,ts,yaml,yml}' --ignore-path ../../.prettierignore",
+        "lint": "eslint . --ext .ts",
+        "lint:fix": "eslint . --ext .ts --fix",
+        "test": "jest --ci --forceExit --detectOpenHandles --colors",
+        "watch": "tsc --watch"
+    },
+    "files": [
+        "LICENSE",
+        "dist",
+        "!dist/*.map",
+        "!dist/**/*.map"
+    ],
+    "dependencies": {
+        "@sap-ux/logger": "workspace:*",
+        "@sap-ux/mockserver-config-writer": "workspace:*",
```

Comment:
Good point, this is also the reason why I like to move CLI commands to configure out of `@sap/ux-ui5-tooling`. To avoid slowing down the initial install after project generation.

For `@sap-ux/create` I see it in several stages:
- If you using it only occasionally, I guess it is fine if it takes bit longer.
- If you use it more often and want to improve performance, you can install `@sap-ux/create` either in a project or globally and use the CLI command.

In case it grows to an extend these two approaches are not sufficient, I think dynamic loading of modules makes a lot of sense.

Reply:
Sounds like a solid plan.